src/math/fir_hifi5.c

```diff
@@ -85,11 +85,6 @@ void fir_get_lrshifts(struct fir_state_32x16 *fir, int *lshift,
 }
 EXPORT_SYMBOL(fir_get_lrshifts);
 
```

Comment:
Commit msg has "WIP - Currently the MCPS saving.." .... is this intended for merge?

Reply:
Good catch, yes I don't think I can come up with any better version. Maybe in some other HiFi5 configuration the performance is better. I've now updated the commit message.